app/controllers/admin/product_import_controller.rb

```diff
@@ -16,9 +17,44 @@ def import
     @shipping_categories = Spree::ShippingCategory.order('name ASC')
   end
 
-  def save
-    @importer = ProductImporter.new(File.new(params[:filepath]), editable_enterprises, params[:settings])
-    @importer.save_all if @importer.has_valid_entries?
+  # def save
+  #   @importer = ProductImporter.new(File.new(params[:filepath]), spree_current_user, params[:settings])
+  #   @importer.save_all if @importer.has_valid_entries?
+  #   @import_into = params[:settings][:import_into]
+  # end
```

Comment:
please, do not comment it out. If it's not needed we can erase it. Git will keep it if we ever want to recover it.